scss/_patterns_navigation.scss

```diff
@@ -2,6 +2,9 @@
 
 $nav-border-bottom-thickness: 1px;
 
+@mixin vf-navigation-default {
+} //only here to prevent a breaking change;
```

Comment:
This seems like a terrible code smell. Why is this?

Reply:
I was under the impression it is used on ubuntu.com, but I checked and it isn't so I've removed it.